README.md

```diff
@@ -28,4 +28,38 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 
 [![Azure DevOps builds](https://img.shields.io/azure-devops/build/azure-sdk/internal/1372?label=eng%2Fcommon%20sync)](https://dev.azure.com/azure-sdk/internal/_build/latest?definitionId=1372&branchName=master)
 
+## Common Engineering System
```

Comment:
I'm fine with having a mention here but I think it is best to move these detailed docs into a MD file under the "doc" directory (which currently doesn't exist) in this repo. I chose "doc" because it is part of the common repository structure policy https://azure.github.io/azure-sdk/policies_repostructure.html.

Reply:
i have ti moved to `doc/common/common_engsys.md`